### DISCUSSION TITLE
What Fonts do you use in Hoppscotch

### DISCUSSION THREAD

**Discussion by DanielK1511** [hoppscotch/hoppscotch#1670] [Q&A] [answered]
Hey Guys,
Odd question i know.

What Fonts are you using for the Application?
![Screenshot_9](https://user-images.githubusercontent.com/36012642/119154659-aac42d00-ba52-11eb-890a-a1e0a63d8b6f.png)
looking forward for your reply

**Comment by liyasthomas** [comment, marked as answer]
We use [Poppins](https://fonts.google.com/specimen/Poppins) font.
